#!/usr/bin/env python
PACKAGE = "irl_stereo"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("left", bool_t, 0, "Left Frame", True)
gen.add("right", bool_t, 0, "Right Frame", False)
gen.add("disp", bool_t, 0, "Disparity", True)
gen.add("raw_disp", bool_t, 0, "Raw Disparity", False)
gen.add("filtered", bool_t, 0, "Filtered", True)
exit(gen.generate(PACKAGE, "irl_stereo", "IRLFrames"))
